bigdata/src/main/scala/org/w3/banana/bigdata/BigdataModule.scala

```diff
@@ -0,0 +1,22 @@
+package org.w3.banana.bigdata
+
+import org.w3.banana._
+
+
+class BigdataModule  extends RDFModule with RDFOpsModule  with RecordBinderModule //with  SparqlGraphModule
+{
+
+  override type Rdf = Bigdata
+
+  /**
+   * Config contains config properties and base URI
+   * baseURI is required for many things in bigdata,
+   * http://todo.example/ is taken only because it is used in Bigdata banana-module
+   */
+  implicit val config =  DefaultBigdataConfig
```

Comment:
I have to provide base:String to many methods (like native select and construct in bigdata). Banana-rdf does not have baseStr:String parameter in its query code so I resolve default baseStr from the module. Do you have any other ideas what we can do with mandatory baseStr:String params?